rails/config/secrets.yml

```diff
@@ -1,11 +1,15 @@
 development:
   secret_key_base: 'ee2be32cf0ca9873ad0774f045de9681895760a702b7ab12d2206d8aa28e58715437e9b73116d8444b14b365e4d7dc13ce6c215dddd45b4b9cb4acbe824151c3'
+  secret_token: <%= ENV["SECRET_TOKEN"] %>
```

Comment:
Where is `SECRET_TOKEN` being set in the environment?

Reply:
Thanks for catching that. We'll remove those in the 7.2 PR which should be done soon.